app/helpers/medication_refill_helper.rb

```diff
@@ -3,21 +3,18 @@
 module MedicationRefillHelper
   include CalendarHelper
   # Save refill date to Google calendar
-  # rubocop:disable RescueStandardError
   def save_refill_to_google_calendar(medication)
     return true unless current_user.google_oauth2_enabled? &&
                        new_cal_refill_reminder_needed?(medication)
-
-    begin
-      args = calendar_uploader_params(medication)
-      CalendarUploader.new(args).upload_event
-    rescue
-      return_to_sign_in
-    else
-      true
-    end
+    args = calendar_uploader_params(medication)
+    CalendarUploader.new(args).upload_event
+    true
```

Comment:
is the `true` here necessary? it seems like a bit of a code smell to be explicitly returning `true`

Reply:
@julianguyen ☝️ I share the same sentiments here. What do you think?

Reply:
I don't think so, but if you look at medications_controller.rb, you'll see the use of `return unless save_refill_to_google_calendar(@medication)` in the create and update methods. So I think simply calling save_refill_to_google_calendar without returning works.